data/publiccloud/terraform/azure.tf

```diff
@@ -185,6 +185,7 @@ resource "azurerm_linux_virtual_machine" "openqa-vm" {
     name              = "${var.name}-${element(random_id.service.*.hex, count.index)}-osdisk"
     caching              = "ReadWrite"
     storage_account_type = "Standard_LRS"
+    disk_size_gb         = 30
```

Comment:
where the 30G comes from? what is the default? looking at the gce i would say that it is 10G. Which 30 makes it too much IMO. Can you check?

Reply:
10GB was the default and we saw it's not enough :)

Reply:
Right, decreeing to `20G`.

Reply:
i still see `30`

Reply:
Pushed now. However your comment applies only to GCE and EC2 as in Azure the default and smallest possible boot disk size is *30G*.